_posts/2020-02-25-our-core-values.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Our Core Values 
+date: 2020-02-25 14:30:00
+categories: ["values"]
+author: etagwerker
+
+---
+
+At [Ombu Labs](https://www.ombulabs.com) we have many values that have been key
+to our success. This is an article about five core values that differentiate 
+our company from the rest.
+
+Every team member is expected to follow these values, especially when things get 
+tough. This is a living document: It's open source and open to enhancements. We 
+have been tweaking these values ever since I started the company.
+
+<!--more-->
+
+## Team First
+
+We can't run a successful consultancy if we don’t value and respect every team 
+member. We don't want people who "live to work". Workaholism is toxic and it 
+leads to burnout. We strive to maintain a healthy work/life balance, to "work 
+to live" a great life, and to grow as professionals within the company.
+
+In terms of our communication, we adhere to the contributor covenant v2.0, not 
+just for our open source projects, but also for any interaction we have with 
+team members: [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md)
+
+## Challenging Projects over Profitable Projects
+
+We value interesting challenges more than high-paying contracts. We are in the 
+problem-solving business for real companies. We won't work on _cool-sounding 
+solutions_ looking for a problem that doesn't exist. We will only work with 
+businesses and entrepreneurs who have real problems. The only way to be great 
+problem solvers is to understand and believe in our clients' businesses.
+
+## Talent is Everywhere
+
+The value of our team will always be higher than the sum of the value of each 
+team member. There is value in team synergy and our interactions. In sports, 
+team play beats raw talent every time. This applies equally in our work 
```

Comment:
maybe this should be "team work"  instead of "team play" ?

Reply:
@FionaDL Just addressed this. 👍